src/index.ts

```diff
@@ -14,23 +15,25 @@ export type Lens<T, R> = {
 }
 export type Scope<T, R> = string | number | Lens<T, R>;
 
-export function pick(selector: Selector | string) {
+export function pick<T>(selector: Selector | string) {
```

Comment:
Hi @goodmind, thanks for the PR.
Could you revert the changes in this part? I know it makes the typings generic for any stream library, but I don't want to make these typings less useful for xstream users. I think the approach should be like in Cycle DOM or Cycle HTTP, where RxJS users import `rxjs-typings.d.ts`. If that's not possible, then RxJS and most.js users should just use `any` casting.